### PR TITLE
VIH-10350 - Decommission obsolete feature flags

### PR DESCRIPTION
**Pull request by marcogagliardi**
### Jira link (if applicable)

[VIH-10350](https://tools.hmcts.net/jira/browse/VIH-10350)

### Change description ###
removed feature toggle service 

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: charts/vh-notification-api/values.dev.template.yaml [hunk 1]
```diff
@@ -4,5 +4,5 @@ java:
   ingressHost: ${SERVICE_FQDN}
   releaseNameOverride: ${RELEASE_NAME}
   environment:
-    SERVICES__NOTIFICATIONAPIRESOURCEID: http://vh-notification-api.dev.platform.hmcts.net
+    SERVICES__NOTIFICATIONAPIRESOURCEID: http://vh-notification-api-pr-159.dev.platform.hmcts.net
```
Comment (marcogagliardi) [inline charts/vh-notification-api/values.dev.template.yaml:7]: to be removed

File: NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs [hunk 2]
```diff
@@ -183,20 +177,15 @@ public async Task<IActionResult> SendParticipantWelcomeEmailAsync(NewUserWelcome
...
+                RoleNames.Individual => NotificationType.ExistingUserLipConfirmationMultiDay,
+                RoleNames.Representative => NotificationType.ExistingUserRepresentativeConfirmationMultiDay,
                 RoleNames.JudicialOfficeHolder when !request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationJohMultiDay,
                 RoleNames.JudicialOfficeHolder when request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationEJudJohMultiDay,
-                RoleNames.Judge when !request.Username.IsJudiciaryUsername() => NotificationType
-                    .HearingConfirmationJudgeMultiDay,
-                RoleNames.Judge when request.Username.IsJudiciaryUsername() => NotificationType
+                RoleNames.Judge => NotificationType
```
Comment (shaed-parkar) [inline NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs:188]: I don't think `HearingConfirmationJudgeMultiDay` is behind a feature toggle but rather `request.Username.IsJudiciaryUsername()`

File: NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs [hunk 3]
```diff
@@ -148,19 +146,15 @@ public async Task<IActionResult> SendParticipantWelcomeEmailAsync(NewUserWelcome
...
-                RoleNames.Representative when useNewTemplates => NotificationType.ExistingUserRepresentativeConfirmation,
+                RoleNames.Individual => NotificationType.ExistingUserLipConfirmation,
+                RoleNames.Representative => NotificationType.ExistingUserRepresentativeConfirmation,
                 RoleNames.JudicialOfficeHolder when !request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationJoh,
                 RoleNames.JudicialOfficeHolder when request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationEJudJoh,
-                RoleNames.Judge when !request.Username.IsJudiciaryUsername() => NotificationType.HearingConfirmationJudge,
-                RoleNames.Judge when request.Username.IsJudiciaryUsername() => NotificationType.HearingConfirmationEJudJudge,
+                RoleNames.Judge => NotificationType.HearingConfirmationEJudJudge,
```
Comment (shaed-parkar) [inline NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs:157]: I don't think `NotificationType.HearingConfirmationJudge` is behind a feature toggle but rather `request.Username.IsJudiciaryUsername()`

File: NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs (hunk 2 above)
Comment (marcogagliardi) [reply NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs:188]: reinstated

File: NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs (hunk 3 above)
Comment (marcogagliardi) [reply NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs:157]: reinstated

**Review by will-craig — APPROVED** [review]

**Review by ronaldmalone — APPROVED** [review]

**Review by shaed-parkar — APPROVED** [review]

**Pull request merged**